lisa/tests/scheduler/load_tracking.py

```diff
@@ -309,9 +309,8 @@ def _test_task_signal(self, signal_name, allowed_error_pct,
 
         signal_mean = area_under_curve(signal) / (signal.index[-1] - signal.index[0])
 
-        if signal_name == 'load':
-            # Load isn't CPU invariant
-            exp_signal /= (self.plat_info['cpu-capacities'][cpu] / UTIL_SCALE)
+        # Since load is now CPU invariant, we don't rescale it back
```

Comment:
Don't we want to guard this against a kernel version check, or did I misunderstand the last discussion we had about this?

At the very least we could have a `logger.warning()` that warns about kernel version < 5.0

Reply:
@ionela-voinescu was more keen on just supporting the current mainline kernel. I don't mind either.

Reply:
I don't believe checking for the kernel version is always safe. It's possible people might backport these patches to older ( < 5.0 ) android kernels so at least controlling test functionality (cpu scaled or not scaled load) based on kernel version seems wrong to me. But I doubt newer kernels would be missing this functionality so it seems right to me to warn people that use older kernels that our estimation of load can be wrong.

Reply:
Ok, let's put a warning if version < 5.0 and unconditionally use the new computation

Reply:
This was introduced in 5.1 I believe